infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/StatusLogger.java

```diff
@@ -195,4 +197,40 @@ public void performance(final String performance) {
   public void warnWeakSubjectivityChecksSuppressed(final UInt64 untilEpoch) {
     log.warn("Suppressing weak subjectivity errors until epoch {}", untilEpoch);
   }
+
+  public void warnWeakSubjectivityFinalizedCheckpointValidationDeferred(
+      final UInt64 finalizedEpoch, final UInt64 wsCheckpointEpoch) {
+    log.warn(
+        "Deferring weak subjectivity checks for finalized checkpoint at epoch {}.  Checks will resume once weak subjectivity checkpoint at epoch {} is reached.",
+        finalizedEpoch,
+        wsCheckpointEpoch);
+  }
+
+  public void finalizedCheckpointOutsideOfWeakSubjectivityPeriod(
+      Level level, final UInt64 latestFinalizedCheckpointEpoch) {
+    log.log(
+        level,
+        "The latest finalized checkpoint at epoch {} is outside of the weak subjectivity period.  Please supply a recent weak subjectivity checkpoint using --ws-checkpoint=<BLOCK_ROOT>:<EPOCH>.",
+        latestFinalizedCheckpointEpoch);
```

Comment:
I'd suggest using `ColorConsolePrinter.print` to make this coloured (I think it should be red at error level and yellow at warn but tempted to just always make it red).

Reply:
good idea 👍

Reply:
updated to log the main errors (inconsistent ws checkpoint, outside ws period) in red and the warnings (deferring / suppressing ws checks due to CLI flags) in yellow